.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,14 @@
+name: "CodeQL config"
+
+query-filters:
+  - exclude:
+      problem.severity:
+        - warning
+        - recommendation
+  - exclude:
+      id: go/log-injection
+
+paths-ignore:
+  - templates
+  - tests
+  - 'api4/*_local.go'
```

Comment:
Are these ignored because local paths have more privilege than other API paths?

Reply:
Yes that's right. These APIs runs only when LOCAL_MODE is enabled and often ends up being false positive.